### PR TITLE
LIVE-1620 Chart updated and ChartCard created

### PR DESCRIPTION
**Pull request by cgrellard-ledger**
[LIVE-1620]

Chart Component Modified

ChartCard Component added

[LIVE-1620]: https://ledgerhq.atlassian.net/browse/LIVE-1620?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by elbywan** [comment]
> ChartCard Component added

⚠️ I'm not sure that this component has been properly exported in the root `index.ts` file.

**Comment by cgrellard-ledger** [comment]
> 

True, c'est fixé

**Comment by cgrellard-ledger** [comment]
> ⚠️ I'm not sure that this component has been properly re-exported in the root index.ts file.

True, c'est fixé

**Review by LFBarreto — APPROVED** [review]

File: packages/native/src/components/ChartCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useMemo } from "react";
+import { useTheme } from "styled-components/native";
+import { Flex, GraphTabs, InfiniteLoader } from "../index";
+import * as Animatable from "react-native-animatable";
```
Comment (elbywan) [inline packages/native/src/components/ChartCard/index.tsx:4]: @LFBarreto @cgrellard-ledger Hey guys, just realized that this `react-native-animatable` has **not** been added to the dependencies in the package.json file so it works only by sheer luck. 

Could one of you make a follow up pr to fix this please? 🙇

**Pull request merged**